libs/blockscout-service-launcher/src/metrics.rs

```diff
@@ -11,12 +11,13 @@ pub struct Metrics {
 impl Metrics {
     pub fn new(service_name: &str, endpoint: &str) -> Self {
         let registry = prometheus::default_registry();
-        let metrics_middleware =
-            PrometheusMetricsBuilder::new(&(service_name.to_owned() + "_metrics"))
-                .registry(registry.clone())
-                .endpoint(endpoint)
-                .build()
-                .unwrap();
+        let const_labels = HashMap::from([("rust_service".into(), service_name.into())]);
```

Comment:
Nit: maybe just:

```suggestion
        let const_labels = HashMap::from([("service".into(), service_name.into())]);
```
?

Reply:
Prometheus will automatically add "service" label equal to kubernetes label, so it's better to have different name

Reply:
Get it. Maybe `service_name` in that case? I just don't like that we use `rust_` prefix in both the metric's name and the label